parser.y

```diff
@@ -1562,13 +1622,13 @@ PlacementOptions:
 	}
 
 PlacementSpec:
-	"ADD" "PLACEMENT" "POLICY" PlacementOptions
+	"ADD" "PLACEMENT" "POLICY" OldPlacementOptions
 	{
 		spec := $4.(*ast.PlacementSpec)
 		spec.Tp = ast.PlacementAdd
 		$$ = spec
 	}
-|	"ALTER" "PLACEMENT" "POLICY" PlacementOptions
+|	"ALTER" "PLACEMENT" "POLICY" OldPlacementOptions
 	{
 		spec := $4.(*ast.PlacementSpec)
```

Comment:
Should we create an issue to track removing the `OldPlacementOptions`? I assume it will be a good first issue.

Reply:
I've added it in the todo of pingap/tidb#26581 :)

There will be a conflict between the implementation. Once the new syntax is implemented, the old syntax must be removed.